pom.xml

```diff
@@ -224,7 +224,7 @@
     <sonar.host.url>https://sonarcloud.io</sonar.host.url>
     <sonar.organization>nuxeo</sonar.organization>
     <sonar.dynamicAnalysis>reuseReports</sonar.dynamicAnalysis>
-    <it.jacoco.destFile>${session.executionRootDirectory}/target/jacoco-it.dump</it.jacoco.destFile>
+    <it.jacoco.destFile>${session.executionRootDirectory}/target/jacoco-it.exec</it.jacoco.destFile>
```

Comment:
It seems that `jacoco-it.exec` is the default option of [`destFile`](http://www.eclemma.org/jacoco/trunk/doc/prepare-agent-integration-mojo.html#destFile)  in goal prepare-agent-integration.

Reply:
Yes, thus the renaming for being consistent. There's no risk of unwanted collision, per contra.